php/commands/src/CLI_Command.php

```diff
@@ -357,7 +357,7 @@ public function update( $_, $assoc_args ) {
 
 		$allow_root = WP_CLI::get_runner()->config['allow-root'] ? '--allow-root' : '';
 		$php_binary = Utils\get_php_binary();
-		$process    = Process::create( "{$php_binary} $temp --info {$allow_root}" );
+		$process    = Process::create( Utils\esc_cmd( '%s %s --info %s', $php_binary, $temp, $allow_root ) );
```

Comment:
I think I'm rethinking my original suggestion to use `Utils\esc_cmd()`, as the originally proposed `escapeshellarg( Utils\get_php_binary() );` covers less surface area.

Thoughts?